extensions/functions_rounding.yaml

```diff
@@ -268,3 +268,43 @@ scalar_functions:
               AWAY_FROM_ZERO, TIE_DOWN, TIE_UP, TIE_TOWARDS_ZERO, TIE_TO_ODD ]
         nullability: DECLARED_OUTPUT
         return: fp64?
+      - args:
+          - value: decimal<P,S>
+            name: "x"
+            description: >
+              Numerical expression to be rounded.
+          - value: i32
+            name: "s"
+            description: >
+              Number of decimal places to be rounded to.
+
+              When `s` is a positive number, the rounding
+              is performed to a `s` number of decimal places.
+
+              When `s` is a negative number, the rounding is
+              performed to the left side of the decimal point
+              as specified by `s`.
+        options:
+          rounding:
+            description: >
+              When a boundary is computed to lie somewhere between two values,
+              and this value cannot be exactly represented, this specifies how
+              to round it.
+
+                - TIE_TO_EVEN: round to nearest value; if exactly halfway, tie
```

Comment:
Can this happen with decimal representations?  I'd argue all of the floating point handling stuff here does not apply.

Reply:
Why would these not apply?  For example, the input value `2.5` could be represented exactly in a decimal type, but rounding it to the nearest integer would result in `2` if the rounding mode is `TIE_TO_EVEN` or `3` if the mode is `TIE_AWAY_FROM_ZERO`.